File: 75-sort-colors/sort-colors.cs
public class Solution {
    public void SortColors(int[] nums) {
       int mid=0;
       int low =0;
       int high = nums.Length-1;

       while(mid <= high)
       {
            if(nums[mid] == 0)
            {
                int value = nums[low];
                nums[low] =  nums[mid];
                nums[mid] = value;
                ++low;
                if(mid <= low)
                {
                    mid++;
                }
            }
            else if(nums[mid] ==1)
            {
                    mid++;
            }
            else
            {
                int value = nums[high];
                nums[high]=nums[mid];
                nums[mid] = value;
                high--;
            }
       }
    }
}

File: 75-sort-colors/sort-colors.py
class Solution:
    def sortColors(self, nums: List[int]) -> None:
        i=0
        j=0
        k=len(nums)-1

        while j<=k:
            if nums[j]==2:
                temp=nums[j]
                nums[j]=nums[k]
                nums[k]=temp
                k-=1
            elif nums[j]==0:
                temp=nums[j]
                nums[j]=nums[i]
                nums[i]=temp
                i+=1
                j+=1
            else:
                j+=1
